lib/logstash/environment.rb

```diff
@@ -76,10 +76,6 @@ def jruby?
       @jruby ||= !!(RUBY_PLATFORM == "java")
     end
 
-    def windows?
-      Gem.win_platform?
-    end
```

Comment:
my concern here is that we end up with inconsistencies between `bootstrap/environment.rb` and `lib/logstash/environment.rb` - we have `windows?` in one place and `jruby?` in the other place